prompb/types.proto

```diff
@@ -56,3 +57,25 @@ message ReadHints {
   int64 start_ms = 3; // Start time in milliseconds.
   int64 end_ms = 4;   // End time in milliseconds.
 }
+
+// Chunk represents a TSDB chunk.
```

Comment:
open/closed should be documented here

Reply:
I guess you mean something like `// Time range [min, max] is inclusive.`?